### PR TITLE
More argo mappings.

### PR DESCRIPTION
**Pull request by Blayyke**

### CODE REVIEW

File: mappings/net/minecraft/value/JsonUtils.mapping [hunk 1]
```diff
@@ -3,7 +3,7 @@ CLASS ol net/minecraft/value/JsonUtils
 	METHOD a createArray (Ljava/lang/Iterable;)Lqe;
 	METHOD a createString (Ljava/lang/String;)Lqa;
 	METHOD a createObject (Ljava/util/Map;)Lqe;
-	METHOD a createArray ([Lgu;)Lqe;
+	METHOD a aJsonArray ([Lgu;)Lqe;
```
Comment (valoeghese) [inline mappings/net/minecraft/value/JsonUtils.mapping]: is it really called "aJsonArray"

Comment (Blayyke) [reply mappings/net/minecraft/value/JsonUtils.mapping]: yes, other methods are also similarly named but i havent renamed them yet

**Comment by Blayyke** [comment]
Don't merge this just yet

File: mappings/net/minecraft/value/JsonUtils.mapping (hunk 1 above)
Comment (Blayyke) [reply mappings/net/minecraft/value/JsonUtils.mapping]: or maybe its not? thoguht it was

Comment (Blayyke) [reply mappings/net/minecraft/value/JsonUtils.mapping]: wait yes it was i am dumb ignore that.

**Comment by Blayyke** [comment]
Okay I just did the rest of the class you commented on : https://github.com/yongjhih/argo/blob/91bd563aee6465e1d9f9b25d75de00fdc0433dee/src/main/argo/jdom/JsonNodeFactories.java

**Comment by valoeghese** [comment]
I can't question the argo names themselves since it's already a library, so I guess I'll merge.

**Pull request merged**